From: =?utf-8?b?Ik1hcmMgRGVxdcOobmVzIChEdWNrKSI=?= <Duck@DuckCorp.org>
Date: Sat, 4 Nov 2023 10:34:21 +0200
Subject: Add -h/--help support to python-argcomplete-check-easy-install-script
Forwarded: no
--- a/argcomplete/scripts/python_argcomplete_check_easy_install_script.py
+++ b/argcomplete/scripts/python_argcomplete_check_easy_install_script.py
@@ -20,7 +20,7 @@
 
 
 def main():
-    if len(sys.argv) != 2:
+    if len(sys.argv) != 2 or sys.argv[1] in ["-h", "--help"]:
         sys.exit(__doc__)
 
     sys.tracebacklimit = 0
--- a/test/test.py
+++ b/test/test.py
@@ -1383,7 +1383,7 @@
             self.sh.run_command("cd " + os.getcwd())
             self.sh.run_command("export PATH=$PATH:./bin")
             self.sh.run_command("export PYTHONPATH=.:$PYTHONPATH")
-            self.sh.run_command('echo -e "[install]\nprefix=" > ~/.pydistutils.cfg')
+            self.sh.run_command('echo -e "[install]\\nprefix=" > ~/.pydistutils.cfg')
             test_package = os.path.join(TEST_DIR, "test_package")
             command = "pip3 install {} --target .".format(test_package)
             if not wheel:
